src/internal/general.ts

```diff
@@ -130,3 +130,19 @@ export async function queryIndexer<T>(args: {
   });
   return data;
 }
+
+export async function getIndexerLastSuccessVersion(args: { aptosConfig: AptosConfig }): Promise<number> {
+  const { aptosConfig } = args;
+
+  const graphqlQuery = {
+    query: GetProcessorStatus,
+  };
+
+  const data = await queryIndexer<GetProcessorStatusQuery>({
+    aptosConfig,
+    query: graphqlQuery,
+    originMethod: "getIndexerLastSuccessVersion",
+  });
+
+  return data.processor_status[0].last_success_version;
```

Comment:
this is the design we follow please make sure you follow it also
https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/types/indexer.ts#L45
https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/internal/account.ts#L456

Reply:
Yep will update to following:

internal/general.ts
Update the use Promise\<GetProcessorStatusLastSuccessResponse\> from `export type GetProcessorStatusLastSuccessResponse = GetProcessorStatusQuery["processor_status"][0];`

api/general.ts
Keep the Promise<number> - Because this is suppose the return the latest version number

Reply:
Actually never mind. Will create 2 different reponse `GetProcessorStatusResponse` and `GetIndexerLastSuccessVersionResponse`